sections/00_title_authors.md

```diff
@@ -1,5 +1,54 @@
 # Opportunities and obstacles for deep learning in biology and medicine
 
-*Randomly-ordered authors here* <sup>*</sup>
+Travers Ching<sup>1,*</sup>, Daniel S. Himmelstein<sup>2</sup>, Brett K.
+Beaulieu-Jones<sup>3</sup>, Alexandr A. Kalinin<sup>4</sup>, Brian T.
+Do<sup>5</sup>, Gregory P. Way<sup>2</sup>, Enrico Ferrero<sup>6</sup>,
+Paul-Michael Agapow<sup>7</sup>, Wei Xie<sup>8</sup>, Gail L. Rosen<sup>9</sup>,
+Benjamin J. Lengerich<sup>10</sup>, Johnny Israeli<sup>11</sup>, Jack
+Lanchantin<sup>12</sup>, Stephen Woloszynek<sup>13</sup>, Anne E.
+Carpenter<sup>14</sup>, Avanti Shrikumar<sup>15</sup>, Jinbo Xu<sup>16</sup>,
+Evan M. Cofer<sup>17</sup>, David J. Harris<sup>18</sup>, Dave
+DeCaprio<sup>19</sup>, Yanjun Qi<sup>12</sup>, Anshul Kundaje<sup>20</sup>,
+Yifan Peng<sup>21</sup>, Laura K. Wiley<sup>22</sup>, Marwin H.S.
+Segler<sup>23</sup>, Anthony Gitter<sup>24,+</sup>, Casey S.
+Greene<sup>2,+</sup>
 
 <sup>*</sup> Author order was determined with a randomized algorithm
+
+<sup>+</sup> To whom correspondence should be addressed: gitter@biostat.wisc.edu
+(A.G.) and csgreene@upenn.edu (C.S.G.)
+
+<sup>1</sup>Molecular Biosciences and Bioengineering Graduate Program,
```

Comment:
I'd recommend doing one subscript per line (as well as one author per line), rather than the wordwrap. You're setting up for some nasty conflicts when authorship updates.